F1WM.UnitTests/Controllers/StandingsControllerTests.cs

```diff
@@ -54,5 +60,74 @@ public async Task ShouldReturnDriversStandingsBySeasonId()
 
 			serviceMock.Verify(s => s.GetDriversStandings(count, seasonId), Times.Once);
 		}
+
+		[Fact]
+		public async Task ShouldReturnCachedConstructorsStandings()
+		{
+			int seasonId = 997;
+			int count = 100;
+			var cacheKey = $"{CacheKeys.ConstructorsStanding}_{seasonId}_{count}";
+
+			cachingServiceMock.Setup(c => c.TryGetCacheValue<ConstructorsStandings>(cacheKey)).Returns(new ConstructorsStandings());
+
+			var result = await controller.GetConstructorsStandings(seasonId, count);
+
+			serviceMock.Verify(s => s.GetConstructorsStandings(count, seasonId), Times.Never);
+			cachingServiceMock.Verify(c => c.TryGetCacheValue<ConstructorsStandings>(cacheKey), Times.Once);
+			cachingServiceMock.Verify(c => c.Set(cacheKey, It.IsAny<ConstructorsStandings>(), TimeSpan.FromDays(1)), Times.Never);
+			var okResult = Assert.IsType<OkObjectResult>(result.Result);
+		}
+
+		[Fact]
+		public async Task ShouldReturnCachedDriversStandings()
+		{
+			int seasonId = 997;
+			int count = 100;
+			var cacheKey = $"{CacheKeys.DriversStanding}_{seasonId}_{count}";
+
+			cachingServiceMock.Setup(c => c.TryGetCacheValue<DriversStandings>(cacheKey)).Returns(new DriversStandings());
+
+			var result = await controller.GetDriversStandings(seasonId, count);
+
+			serviceMock.Verify(s => s.GetDriversStandings(count, seasonId), Times.Never);
+			cachingServiceMock.Verify(c => c.TryGetCacheValue<DriversStandings>(cacheKey), Times.Once);
+			cachingServiceMock.Verify(c => c.Set(cacheKey, It.IsAny<DriversStandings>(), TimeSpan.FromDays(1)), Times.Never);
```

Comment:
We should make sure that `Set` isn't called with any param, so concrete `TimeSpan` should be changed to fitting `It.IsAny` setup